tmail-backend/mailets/src/main/java/com/linagora/tmail/mailet/LDAPMailingList.java

```diff
@@ -65,6 +67,10 @@
  *  <li>userMailCacheDuration: Time during which one should keep entries into the user DN => mailAddress cache.</li>
  *  <li>mailAttributeForGroups: Attribute holding the mail address of a group. For easy testing this can be set to description
  *  but for production use a special LDAP schema needs to be crafted for using the mail attribute.</li>
+ *  <li>businessCategoryExtractAttribute: Optional string. Default to none. If configured, businessCategory would be
```

Comment:
Not fan of the extra attribute to be honnest, I'd prefer we always move on with sanitizing.

This would simplify the configuration.

Reply:
OK.

But I guess we can not do the same on James side.

Reply:
IMO we can.

Reply:
> Not fan of the extra attribute to be honnest, I'd prefer we always move on with sanitizing.

Do you want both normal and sanitize mode? Or only sanitize mode?

BTW on James side we can not just hardcode to sanitize the `cn`, no?

Reply:
> BTW on James side we can not just hardcode to sanitize the cn, no?

matcher configuration is hard TBH as everything has to fit in one string.

We could have a system variable to disable that.

Reply:
> Do you want both normal and sanitize mode? Or only sanitize mode?

Both.

I have a piece of code adressing my comments. Just running tests and I will push it.